cli/cmd/integration.go

```diff
@@ -103,6 +106,16 @@ var (
 				return errors.New(msg)
 			}
 
+			integrationType, _ := api.FindIntegrationType(integration.Data[0].Type)
+			err = cli.LwApi.V2.Schemas.GetService(integrationType.Schema()).Get(args[0], &resp)
+
+			if err != nil {
+				cli.Log.Debugw("unable to get integration service")
+			}
+
+			if resp.Data.State != nil {
+				integration.Data[0].State.Details = resp.Data.State.Details
```

Comment:
if you are accessing `integration.Data[0]`, we need to check that it is not empty, if not, the CLI will panic

Reply:
We check that integration.Data is empty on line 102 and and error is returned if it is.